HACKING.md

```diff
@@ -0,0 +1,44 @@
+# Notes for hacking on local-storage-operator
+
+## For developing on plain k8s
+
+Developing on plain k8s is pretty simple. Just write your code and use Makefile to build your own images
+and then update `deploy/operator.yaml` to point to your images and follow rest of the instructions in `deploy/README.md`.
+
+## For developing on Openshift and OLM
+
+1. Same as plain k8s, make your code change and build local-diskmaker and local-storage-operator images. Push them to quay or docker.io.
+2. Next we will have to update CSV inside manifests directory to point to those images.
+3. After updating the manifests file, you need to build your own local-registry. You can use `Dockerfile.registry` to do that.
+
+```
+docker build --no-cache -t quay.io/gnufied/local-registry:latest -f ./Dockerfile.registry
```

Comment:
can't this been done with buildah/podman ?

Reply:
I am using podman.. and these instructions work identical for podman too.